### PR TITLE
fix bug, infinite recursive call of GetLoadBalancerName()

### LINKED ISSUE #68337: Infinite recursive call
/kind bug
/sig release

Such as:
https://github.com/kubernetes/kubernetes/blob/4bc9e94feec17d1980e733a08699255482d15f4e/pkg/cloudprovider/providers/cloudstack/cloudstack_loadbalancer.go#L242
https://github.com/kubernetes/kubernetes/blob/4bc9e94feec17d1980e733a08699255482d15f4e/pkg/client/chaosclient/chaosclient.go#L66

### PR DESCRIPTION
**Pull request by wgliang**
**What this PR does / why we need it**:
fix bug, infinite recursive call of GetLoadBalancerName()

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #68337

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by wgliang** [comment]
PTAL
@kubernetes/sig-cloud-provider-pr-reviews

File: pkg/cloudprovider/providers/cloudstack/cloudstack_loadbalancer.go [hunk 1]
```diff
@@ -23,7 +23,9 @@ import (
 
 	"github.com/golang/glog"
 	"github.com/xanzy/go-cloudstack/cloudstack"
+
 	"k8s.io/api/core/v1"
+	"k8s.io/kubernetes/pkg/cloudprovider"
```
Comment (andrewsykim) [inline pkg/cloudprovider/providers/cloudstack/cloudstack_loadbalancer.go]: This should be 
```go
cloudprovider k8s.io/cloud-provider
```

Comment (wgliang) [reply pkg/cloudprovider/providers/cloudstack/cloudstack_loadbalancer.go]: This is a new repo? I can not see any others packages import it.

Comment (andrewsykim) [reply pkg/cloudprovider/providers/cloudstack/cloudstack_loadbalancer.go]: Yeah there's quite of a few now, maybe you need to pull latest master?

Comment (wgliang) [reply pkg/cloudprovider/providers/cloudstack/cloudstack_loadbalancer.go]: Done.

**Comment by hogepodge** [comment]
Solid and simple fix.

/lgtm

**Comment by andrewsykim** [comment]
/approve 

sorry for the delay, thank you @wgliang

**Comment by ustcweizhou** [comment]
@wgliang @k8s-ci-robot 
can it be merged into 1.12 as well ?

**Comment by andrewsykim** [comment]
Yes, I think this should be cherry-picked

**Comment by wgliang** [comment]
@andrewsykim @ustcweizhou I will cherry-picke it.

**Pull request merged**